apps/awsapps/s3.go

```diff
@@ -0,0 +1,39 @@
+package awsapps
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+)
+
+const (
+	// S3BucketEnvVar is the environment variable containing the S3 bucket name
+	// used to host Apps' assets.
+	S3BucketEnvVar = "MM_APPS_S3_BUCKET"
+
+	// DefaultS3Bucket is the default S3 bucket name.
+	DefaultS3Bucket = "mattermost-apps-bucket"
```

Comment:
Since S3 bucker names are globally unique, I wonder if we need a reserved name for the "default" MM Cloud one and a different one for the non-MM AWS Cloud ones (not sure if we can include some client-specific info here).

Reply:
Right. I wonder if we should get rid of the default altogether since it is specific to MM Cloud.